workshop/exercise-3/README.md

```diff
@@ -88,15 +88,11 @@ The file should look something like:
 	```
 6. You should see the first 20 Fibonacci numbers!
 
-7. If we left this application alone for some time, it would scale itself back down to 0, and terminate the pods that were created. The default for Knative scale-to-zero is 5 minutes. Let's decrease this time by editing the autoscaler:
+7. If we left this application alone for some time, it will scale itself back down to 0, and terminate the pods that were created.
 
-	```
-	kubectl edit cm config-autoscaler --namespace knative-serving
```

Comment:
IMO we should wait until we update the knative version.

Reply:
Let's update the lab for v0.3.0 now then